evennia/contrib/worldloader/example_tutorial_world/worlddata/models.py

```diff
@@ -0,0 +1,125 @@
+from django.db import models
+
+
+#------------------------------------------------------------
+#
+# store all rooms
+#
+#------------------------------------------------------------
+class world_rooms(models.Model):
+    "Store all unique rooms."
+
+    key = models.CharField(max_length=255, primary_key=True)
+    name = models.CharField(max_length=255)
+    alias = models.CharField(max_length=255, blank=True)
+    typeclass = models.CharField(max_length=255)
+    desc = models.TextField(blank=True)
+    location = models.CharField(max_length=255, blank=True)
+    home = models.CharField(max_length=255, blank=True)
+    lock = models.CharField(max_length=255, blank=True)
+    attributes = models.TextField(blank=True)
+    tutorial_info = models.TextField(blank=True)
```

Comment:
Data in example_tutorial_world are used for tutorial world only. If someone want to create his own game world, he can modify these models. The models in example_empty_world haven't ```tutorial_info```.